### PR TITLE
FIX Make table row click open popup, make trigger keyboard accessible

### PR DESCRIPTION
**Pull request by robbieaverill**
Follow up for #68

image

cc @clarkepaul

### CODE REVIEW

File: client/src/components/ModuleDetails/ModuleDetails.js [hunk 1]
```diff
@@ -19,10 +19,13 @@ class ModuleDetails extends Component {
   /**
    * Toggle the popover with module details being open or not
    */
-  toggle() {
+  toggle(e) {
```
Comment (NightJar) [inline client/src/components/ModuleDetails/ModuleDetails.js]: Could this be a full `event`?

Comment (robbieaverill) [reply client/src/components/ModuleDetails/ModuleDetails.js]: Sure

**Comment by robbieaverill** [comment]
Its updated

**Comment by NightJar** [comment]
Ideal, thanks :)

**Pull request merged**